ngraph_bridge/ngraph_encapsulate_impl.h

```diff
@@ -255,6 +246,12 @@ class NGraphEncapsulateImpl {
       m_executable_pipelined_tensors_map;
 
   int m_depth{2};  // TODO make this settable
+
```

Comment:
These functions are no longer directly used by encapsulate op, so moved them to private